qrec/stage_run.py

```diff
@@ -1,216 +1,82 @@
 """
 Stage run
 
-The objective of this script is to run the experiment,
-track he hyperparameters, save the values of interest
-and make the decisions.
+The functions of this module implement the reinforced learning protocol over the simulated device.

```

Comment:
I changed the name of the module to use only lowercase letters to accomplish the standards.